extend azure docs (#371)
defang-io/defang-docs: docs/concepts/managed-llms/managed-language-models.md

# Managed Language Models

> Each cloud provider offers their own managed Large Language Model services. AWS offers Bedrock, GCP offers Vertex AI, Azure offers Azure OpenAI Service, and Digital Ocean offers their GenAI platform. Defang makes it easy to leverage these services in your projects.

Reviewer: We are using [Microsoft Foundry](https://azure.microsoft.com/en-us/products/ai-foundry)

Author: ```suggestion
Each cloud provider offers their own managed Large Language Model services. AWS offers Bedrock, GCP offers Vertex AI, Azure offers Microsoft Foundry, and Digital Ocean offers their GenAI platform. Defang makes it easy to leverage these services in your projects.
```

## Current Support

| Provider | Managed Language Models |
| --- | --- |
| [Playground](/docs/providers/playground#managed-llms) | ✅ |
| [AWS Bedrock](/docs/providers/aws#managed-llms) | ✅ |

> | [Azure OpenAI Service](/docs/providers/azure) | ❌ |

Author: ```suggestion
| [Azure OpenAI Service](/docs/providers/azure) | ✅ |
```

> | [Azure OpenAI Service](/docs/providers/azure) | ✅ |

Author: ```suggestion
| [Microsoft Foundry](/docs/providers/azure) | ✅ |
```
